Assets/Scripts/Data/Core/Database.cs

```diff
@@ -22,18 +22,44 @@ public static T Instance
 
 	#endregion
 
+	public bool IsInitialized
+	{
+		get
+		{
+			return this.isInitialized;
+		}
```

Comment:
Felt it was more expressive, but happy to define a convention for use of keyword if we like